runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -310,4 +339,98 @@ private void removeGameframe()
+				slayer = client.getWidget(WidgetInfo.SKILL_SLAYER);
+
+		membersOnlyWidgets.add(agility);
+		membersOnlyWidgets.add(construction);
+		membersOnlyWidgets.add(farming);
+		membersOnlyWidgets.add(fletching);
+		membersOnlyWidgets.add(herblore);
+		membersOnlyWidgets.add(hunter);
+		membersOnlyWidgets.add(thieving);
+		membersOnlyWidgets.add(slayer);
+
+		return membersOnlyWidgets;
+	}
+
+	private void shuffleSkills(boolean hideMembers)
+	{
+		if (hideMembers && client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(WOODCUTTING_X);
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeY(WOODCUTTING_Y);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeX(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeY(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeX(PRAYER_X);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeY(PRAYER_Y);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeX(client.getWidget(WidgetInfo.SKILL_HERBLORE).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeY(client.getWidget(WidgetInfo.SKILL_HERBLORE).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeX(client.getWidget(WidgetInfo.SKILL_THIEVING).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeY(client.getWidget(WidgetInfo.SKILL_THIEVING).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeX(CRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeY(CRAFTING_Y);
+		}
+		else if (client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(TOTAL_X);
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeY(TOTAL_Y);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeX(PRAYER_X);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeY(PRAYER_Y);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeX(MAGIC_X);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeY(MAGIC_Y);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeX(CRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeY(CRAFTING_Y);
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeX(RUNECRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeY(RUNECRAFTING_Y);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeX(WOODCUTTING_X);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeY(WOODCUTTING_Y);
+		}
+	}
+
+	private void showMemberOnlySkills()
+	{
+		for (Widget widget : getMemberSkills())
+		{
+			if (widget != null)
+			{
+				widget.setHidden(config.hideMembersSkills());
+			}
+		}
+		shuffleSkills(config.hideMembersSkills());
+		updateTotalLevels();
+	}
+
+	private void updateTotalLevels()
```

Comment:
Does this work with the virtual levels plugin?

Reply:
Actually, this probably should not be reading widget values, and should be getting skill levels from client instead.

Reply:
Done.